### DISCUSSION TITLE
New expire option: redirect to another page

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#362] [Feature Requests]
https://wordpress.org/support/topic/revert-to-different-page/

> Can I revert to another Page?
> 
> I have a Jobs page and want to revert a current advert to a different page that says “Sorry but no jobs are available”

**Comment by stevejburge** [comment]
One more request:

> how does it handle redirections as I would ideally like it to show links to currently available posts with similar keywords that the deleted post had in the title. 301 redirection, etc?

**Comment by stevejburge** [comment]
One more:

"I’ve got a killer idea to make Future Pro a needed plugin. Call it ‘redirect on expiration’ or ‘redirect on delete’ There also could be a fallback option to redirect to the category archive page that the post was in."
<img width="972" alt="Screen Shot 2022-11-11 at 1 46 55 PM" src="https://user-images.githubusercontent.com/3868028/201410072-77dfcb5d-0e8c-48cd-9c86-a8a5710e8246.png">

**Comment by stevejburge** [comment]
One more request for this today

**Comment by kaschwei** [comment]
If the expiry date has been reached, it would be good if you could redirect to another page without a message appearing that this page cannot be found or even better would be to be able to place a message on this page that the expiry date has been reached, please contact us to the administrator.

**Comment by andergmartins** [comment]
One more, related to #1550
